MAINTAINERS_GUIDELINES.md

```diff
@@ -1,8 +1,19 @@
-# Release Instructions
+# Guidelines for Maintainers
+
+## Repository Policies
+
+Please follow these principles for this repository:
+
+- pull requests require 2+ approvals
```

Comment:
A little bit off the topic, but it seems that some polices can be enforced by GitHub settings. https://github.com/isaacs/github/issues/1001

Reply:
added 2+ approvals as branch policy in GitHub